### PR TITLE
Intuitionize section "Infinite primes theorem"

### PR DESCRIPTION
**Pull request by jkingdon**
The main thing here which isn't just straightforward intuitionizing is some new theorems about decidable sets of integers, in the same spirit as existing theorems like https://us.metamath.org/ileuni/nnmindc.html or https://us.metamath.org/ileuni/unbendc.html .

### CODE REVIEW

File: iset.mm [hunk 1]
```diff
@@ -132166,6 +132187,142 @@ which defines the set we are taking the supremum of must (a) be true at
...
+      XJXKXNZUXEUXFEUXEUWJKUXEUWJYMUXDVDXOUXEXFVNZUXKXGUXEVAUXFIJZUXJUXDUXMYMUW
+      JXPXQUXEEUXFUXEEUXKVPUXEUXFUXLVPXRXSEUXGYDXTWIUXDUWLUXIWQYMUXDUWKUXHYTUXH
+      FUWJOYQUWJQZYSUXGBUXNYRUXFENYQUWJKLUNRUOYAXLXQVRWTUAUDUEUUAUFYBYCYE $.
+  $}
+
+  ${
+    $d A j v z $.  $d A v w $.  $d v w x y $.
+    nnwof.1 $e |- F/_ x A $.
+    nnwof.2 $e |- F/_ y A $.
+    $( Well-ordering principle: any nonempty set of positive integers has a
```
Comment (avekens) [inline iset.mm:132290]: nonempty -> inhabited?

**Comment by avekens** [comment]
Very interesting results, but I do not see how the new theorems about decidable sets of integers are used/needed to show that there are infinitely many primes...

File: iset.mm (hunk 1 above)
Comment (jkingdon) [reply iset.mm:132290]: Changed to "inhabited decidable".

**Comment by jkingdon** [comment]
> Very interesting results, but I do not see how the new theorems about decidable sets of integers are used/needed to show that there are infinitely many primes...

The usages are:

```
nnwodc is used by uzwodc and nnwofdc
nnwofdc is used by nnwosdc
nnwosdc is used by infpnlem2

uzwodc is not used here but is being intuitionized
simply because it exists in set.mm. 
```

As for why `infpnlem2` wants a theorem about the smallest element of a decidable set of integers, it wants to show that a natural number N has a prime larger than it. Take the set of all divisors of `` ( ! ` N ) + 1 `` which are at least two and take the smallest element.  This number is prime and larger than N.

**Review by avekens — APPROVED** [review]

**Pull request merged**